explorer/app/components/Export/components/ExportToTerra/components/ExportToTerraNotStarted/exportToTerraNotStarted.tsx

```diff
@@ -0,0 +1,40 @@
+import { ButtonPrimary } from "app/components/common/Button/button.styles";
+import { RoundedPaper } from "app/components/common/Paper/paper.styles";
+import {
+  Content as SectionText,
+  Section,
+} from "app/components/Project/components/Section/section.styles";
+import React from "react";
+import { SectionTitle } from "../../../../../Project/components/Section/components/SectionTitle/sectionTitle";
+import { BatchNormalizationWarning } from "../../../BatchNormalizationWarning/batchNormalizationWarning";
+import {
+  Actions as SectionActions,
+  SectionContent,
+} from "./exportToTerraNotStarted.styles";
+
+export type RunFn = () => void;
+
+interface Props {
+  run: RunFn;
+}
+
+export const ExportToTerraNotStarted = ({ run }: Props): JSX.Element => {
+  return (
+    <RoundedPaper>
+      <Section>
+        <SectionContent gap={2}>
+          <SectionTitle title="Select File Types" />
```

Comment:
Is it possible to just have the batch normalization warning and request link button as the section content here?